server/src/api/api.graphql

```diff
@@ -7,6 +7,8 @@ type Query {
 	items: [Item!]
     # Get a list containing each category that is on at least one item
     categories: [Category!]
+	# Get a single request by its ID
```

Comment:
the comment should say retrieve a particular request by relevant search params, no?

Reply:
maybe should support multiple ids, or statuses. Given requests load, I would expect we could support this all in memory, not sure how necessary single search is?

Reply:
That comment is out of date.  It'll request multiple requests so you can search by status and such, although it's still missing the ability to search my multiple statuses, e.g. those that are SUBMITTED or APPROVED.